crates/bevy_ecs/src/schedule/run_criteria.rs

```diff
@@ -457,11 +457,6 @@ impl System for RunOnce {
 
     fn check_change_tick(&mut self, _change_tick: u32) {}
 
-    fn run(&mut self, input: Self::In, world: &mut World) -> Self::Out {
```

Comment:
Out of curiosity, why is this removed?

Reply:
It's predefined as a default, just extra lines if it's included :man_shrugging: